src/components/Navbar.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] As the previous reviewer mentioned kindly consider fixing this linter error before requesting a review